exasol_error_reporting_python/error_message_builder.py

```diff
@@ -0,0 +1,40 @@
+
+class ErrorMessageBuilder:
+    def __init__(self, error_code: str):
+        self._error_code = error_code
+        self._message_builder = []
+        self._mitigations = []
+
+    def message(self, message: str):
```

Comment:
Please also ensure that the ASTs don't get to weird, such that we can still can parse them.

Reply:
I didn't get it how we have to ensure  that the ASTs don't get to weird,.

Reply:
Generate them for examples and look at them and see if you could write an algorithm to extract from the AST

Reply:
cool :+1: